velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -485,6 +485,128 @@ TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
           "year(c0)", std::nullopt, std::nullopt));
 }
 
+TEST_F(DateTimeFunctionsTest, week) {
+  const auto week = [&](StringView date) {
```

Comment:
This seems a time string and not date. Please change the name of the parameter. 

Also should the parameter be const StringView&